web/src/components/timeControls.jsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { useTranslation } from '../helpers/translation';
 import styled, { css } from 'styled-components';
-import { formatHourlyDate } from '../helpers/formatting';
+import { formatDate, formatTimeRange } from '../helpers/formatting';
+import { TIME } from '../helpers/constants';
 
```

Comment:
On yearly, every tick except first and last are off by one 🤔 
<img width="453" alt="Screenshot 2022-06-07 at 14 12 12" src="https://user-images.githubusercontent.com/3296643/172376061-7e8c6f40-54d5-4f74-bdbe-4ba27faab2e5.png">

Reply:
Fixed by using datetimes from app-backend. Right now the app-backend state is a bit off, but will be fixed